### PR TITLE
direnv(feat): Report direnv issues

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

**Comment by armenzg** [comment]
@billyvg no rush. I simply want to get your feedback wrt which of the two PRs you would prefer.

**Comment by billyvg** [comment]
+1 for this if the trade-off is just the traceback

File: .envrc [hunk 1]
```diff
@@ -99,6 +113,17 @@ export NODE_OPTIONS=--max-old-space-size=4096
 # Enable this by default for development envs (CI/deploys do not use envrc)
 export SENTRY_UI_HOT_RELOAD=1
 
+## Initialize sentry-cli
+if ! require sentry-cli; then
+    curl -sL https://sentry.io/get-cli/ | bash
+fi
```
Comment (billyvg) [inline .envrc:119]: Can we install only when we need to? i.e. in `report_to_sentry`

Comment (armenzg) [reply .envrc]: That's fine with me.

**Review by billyvg — APPROVED** [review]

**Pull request merged**